addon/index.js

```diff
@@ -211,18 +212,28 @@ export function changeset(obj, validateFn = defaultValidatorFn, validationMap =
       let content = get(this, CONTENT);
       assert('Cannot merge with a non-changeset', isChangeset(changeset));
       assert('Cannot merge with a changeset of different content', get(changeset, CONTENT) === content);
-      assert('Cannot merge invalid changesets', get(this, 'isValid') && get(changeset, 'isValid'));
 
       if (get(this, 'isPristine') && get(changeset, 'isPristine')) {
         return this;
       }
 
       let changesA = get(this, CHANGES);
       let changesB = get(changeset, CHANGES);
-      let mergedChanges = pureAssign(changesA, changesB);
+
```

Comment:
Do you think you could add a `jcscrc` or `eslint` config with the rules you're after?